packages/suite/src/actions/settings/__tests__/deviceSettingsActions.test.ts

```diff
@@ -22,12 +22,17 @@ jest.mock('trezor-connect', () => {
         setTestFixtures: (f: any) => {
             fixture = f;
         },
+        DEVICE: {
+            CHANGED: 'device-changed',
+        },
     };
 });
 
 export const getInitialState = (): any => {
+    const device1 = getSuiteDevice();
```

Comment:
```suggestion
    const device = getSuiteDevice();
```
why 1?

Reply:
perosnal preference :D cause when I'll add more test with multiple devices inside the reducer it's nicer to have names like device1, device2 instead of device, device2.